core/domain/takeout_service_test.py

```diff
@@ -18,22 +18,28 @@
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
 import datetime
+
 from constants import constants
 from core.domain import exp_domain
 from core.domain import exp_services
 from core.domain import feedback_services
 from core.domain import rights_manager
 from core.domain import takeout_service
+from core.domain import topic_domain
 from core.platform import models
 from core.tests import test_utils
 import feconf
 import utils
 
 (
-    collection_models, email_models, exploration_models, feedback_models,
-    suggestion_models, user_models) = models.Registry.import_models([
-        models.NAMES.collection, models.NAMES.email, models.NAMES.exploration,
-        models.NAMES.feedback, models.NAMES.suggestion, models.NAMES.user])
+    base_models, collection_models, email_models,
+    exploration_models, feedback_models, skill_models, topic_models,
```

Comment:
Keep line breaks here the same as those in lines 39-42, so that the two lists can be easily compared. I.e. break after skill_models and user_models instead.

Reply:
Done.-